.github/workflows/web-ci.yml

```
@@ -0,0 +1,98 @@
+  test-fastapi-backend:
+    name: Test FastAPI Backend (Python)
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./web_app/backend
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
```

actionlint (1.7.11)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)